paasta_tools/ide_helper.py

```diff
@@ -0,0 +1,163 @@
+import json
```

Comment:
hmm, i'd say we should put this script in contrib/, but it seems like paasta_tools/ has already become somewhat of a dumping ground for all sorts of random stuff so i guess we can leave this here until we eventually do some reorganizing :p

Reply:
I added the script in contrib/